versioned_docs/version-v1.1/upgrade/troubleshooting.md

```diff
@@ -135,3 +135,67 @@ The Harvester controller deletes the upgrade repository VM and all files that ar
+
+![The "Enable Logging" checkbox on the upgrade confirmation dialog](/img/v1.1/upgrade/enable_logging.png)
+
+You can click the **Download Log** button to download the log archive during an upgrade.
+
+![Download the upgrade log archive by clicking the "Download Log" button on the upgrade dialog](/img/v1.1/upgrade/download_upgradelog_dialog.png)
+
+Log entries will be collected as files for each upgrade-related Pod, even for intermediate Pods. The support bundle provides a snapshot of the current state of the cluster, including logs and resource manifests, while the upgrade log preserves any logs generated during an upgrade. By combining these two, you can further investigate the issues during upgrades.
+
+![The upgrade log archive contains all the logs generated by the upgrade-related Pods](/img/v1.1/upgrade/upgradelog_archive.png)
+
+After the upgrade ended, Harvester stops collecting the upgrade logs to avoid occupying the disk space. In addition, you can click the **Dismiss it** button to purge the upgrade logs.
+
+![The upgrade log archive contains all the logs generated by the upgrade-related Pods](/img/v1.1/upgrade/dismiss_upgrade_to_remove_upgradelog.png)
+
+For more details, please refer to the [upgrade log HEP](https://github.com/harvester/harvester/blob/master/enhancements/20221207-upgrade-observability.md).
+
+:::caution
+
+The storage volume for storing upgrade-related logs is 1GB by default. If an upgrade went into issues, the logs may consume all the available space of the volume. To work around such kind of incidents, try the following steps:
+
+1. Detach the `log-archive` Volume by scaling down the `fluentd` StatefulSet and `downloader` Deployment.
+
+```
+# Locate the StatefulSet and Deployment
+$ kubectl -n harvester-system get statefulsets -l harvesterhci.io/upgradeLogComponent=aggregator
+NAME                                               READY   AGE
+hvst-upgrade-xxxxx-upgradelog-infra-fluentd   1/1     43s
+
+$ kubectl -n harvester-system get deployments -l harvesterhci.io/upgradeLogComponent=downloader
+NAME                                            READY   UP-TO-DATE   AVAILABLE   AGE
+hvst-upgrade-xxxxx-upgradelog-downloader   1/1     1            1           38s
+
+
+# Scale down the resources to terminate any Pods using the volume
+$ kubectl -n harvester-system scale statefulset hvst-upgrade-xxxxx-upgradelog-infra-fluentd --replicas=0
+statefulset.apps/hvst-upgrade-xxxxx-upgradelog-infra-fluentd scaled
+
+$ kubectl -n harvester-system scale deployment hvst-upgrade-xxxxx-upgradelog-downloader --replicas=0
+deployment.apps/hvst-upgrade-xxxxx-upgradelog-downloader scaled
+```
+
+2. Find out and expand the `log-archive` volume size via the Longhorn dashboard. For more details, please refer to [the volume expansion guide](https://longhorn.io/docs/1.3.2/volumes-and-nodes/expansion/).
+
+```
+# Here's how to find out the actual name of the target volume
```

Comment:
Thanks. We also need to add this to the `docs/upgrade/troubleshooting.md` file.

Reply:
Sorry, I missed that. Updated!